### ISSUE TITLE
Non-indented comments confuse SimpylFold

### ISSUE THREAD

**Issue by GreyGnome** [tmhedberg/SimpylFold#42]
Hi,
If I have a function like this:
```
def my_func():
#This is a comment.
#This too.
#They start in column 1.
    print "Here is a properly indented code snippet."
```
then SimpylFold will not fold my_func(). Not a big deal, I suppose, but Python allows for comments without concern for indentation level (hence I bumped into this phenomenon).

Not a big deal... it's nice to get folding... but some of my comments include examples that are rather long so I prefer to stick them on the lefthand side. Maybe that's generally hated, I don't know.

In any event, thanks for SimpylFold. It's nice.

**Comment by Fruglemonkey** [comment]
Placing a single space at the start of each # will lead to you getting folding as per usual.

![image](https://cloud.githubusercontent.com/assets/7358629/9893812/cc46318a-5c70-11e5-9fb5-3d78fb06b665.png)

**Comment by tmhedberg** [comment]
Thanks for reminding me of this. I've just merged pull request #39 to fix it.

In general, SimpylFold assumes that you roughly follow the PEP 8 formatting conventions, and in this particular case, PEP 8 [states](https://www.python.org/dev/peps/pep-0008/#block-comments): "Block comments generally apply to some (or all) code that follows them, and are indented to the same level as that code." Still, the fix is simple enough that it makes sense to include anyway.

**Comment by GreyGnome** [comment]
Thanks. Yes, I realize that commenting on the lefthand side may not be purty but at the same time there are moments when I'd like to do it (in order to keep line wrapping from occurring). I appreciate your response.  And the plugin works very nicely.
